QModManager/API/IQModServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace QModManager.API
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// An interface of services that other mods can make use of.
+    /// </summary>
+    public interface IQModServices
+    {
+        /// <summary>
+        /// Gets a list successfully loaded mods.
+        /// </summary>
+        /// <returns>
+        /// The loaded mods.
+        /// </returns>
+        List<IQMod> GetLoadedMods();
```

Comment:
Add the following service method:
```CS
IQMod FindMod(string id);  // Returns null if mod does not exist
```

Reply:
With the notice that the method might not work correctly if used in PrePatch

Reply:
> With the notice that the method might not work correctly if used in PrePatch

The method names and summaries were changed and no longer imply "successfully loaded". 

Also, this list is populated _before_ PrePatch methods are invoked.  
There is no way for this to fail, even on the earliest PrePatch.

Consumers of this service can check the `IsLoaded` get property of an `IQMod` to check if it has loaded or not.